### PR TITLE
Seed from geometry in blender

### PR DESCRIPTION
**Pull request by PavelBlend**
I made seed in blender. True, I had a problem that I do not know how to solve:
https://github.com/taichi-dev/taichi_elements/blob/21ea2e19131b8f9c78e93cdb6f148283fe6a5b28/operators.py#L108

### CODE REVIEW

**Comment by PavelBlend** [comment]
Test render in Cycles.
water_suzanne

**Comment by yuanming-hu** [comment]
Looks GREAT!!

File: operators.py [hunk 1]
```diff
@@ -103,6 +105,11 @@ def init_simulation(self):
 
         simulation_class = self.scene.elements_nodes[simulation_node.name]
 
+        # simulation_class.frame_start - not working! Why? To figure out.
+        self.frame_start = simulation_class.params['frame_start']
```
Comment (yuanming-hu) [inline operators.py:109]: Do you mean `simulation_class.frame_end` works but `simulation_class.frame_start` doesn't?

Comment (PavelBlend) [reply operators.py:109]: Yes. To get the value of frame_start you need to write this way:
` self.frame_start = simulation_class.params['frame_start']`
Instead
`self.frame_start = simulation_class.frame_start`

Comment (PavelBlend) [reply operators.py:109]: I found a mistake and fixed it. Now I will `push`.

Comment (yuanming-hu) [reply operators.py:109]: I have no clear idea, but it seems to me the only difference between `frame_end` and `frame_start` is that 
https://github.com/taichi-dev/taichi_elements/blob/bdada022532f1ae5a941c963d947687b609fc8e6/handlers.py#L125
sets `frame_end` but not `frame_start`. Do we also need to set `frame_start` here?

Comment (PavelBlend) [reply operators.py]: fix: 9ae8d45380a174f3a767a719a3509fe8148cecc8

**Comment by yuanming-hu** [comment]
Please feel free to merge this PR when you are ready.

Btw, could you upload the blender file for this simulation & rendering? I'll play with it when I get a chance. Feel free to create a new release titled `sample files` (you have Admin permission to this repo so you can do whatever you want) to hold these files.

**Comment by PavelBlend** [comment]
I think it’s too early to create examples of scenes, since the node system has not yet been formed (I plan to redo it a bit). And examples of scenes quickly become outdated with each new commit (they simply won’t work when I start making changes to the nodes).

Here is the scene:
[test_seeding.zip](https://github.com/taichi-dev/taichi_elements/files/4452680/test_seeding.zip)

**Pull request merged**